HACKING.md

```diff
@@ -182,14 +182,33 @@ git submodules:
 Refer to the [testing README](test/README.md) for details on running the Cockpit
 integration tests locally.
 
-## Testing the Python bridge
+## Python bridge
 
 Cockpit currently has an experimental replacement for `cockpit-bridge`
 written in Python.  It resides in `src/cockpit` with most of its rules in
 `src/Makefile.am`.  This directory was chosen because it matches the standard
 so-called "src layout" convention for Python packages, where each package
 (`cockpit`) is a subdirectory of the `src` directory.
 
+### Running the bridge
+
+A test image with the Python image can be created with `image-prepare` for the operating systems which don't enable it yet.
+
+    ./test/image-prepare --python
+
+Or you can interactively run the bridge locally:
+
+    python3 -m cockpit.bridge
+
+To enable debug logging, you can set `COCKPIT_DEBUG`. In a test virtual
+machine this can be done by setting it in the global environment:
+
+    echo 'COCKPIT_DEBUG=all' >> /etc/environment
+
+Debug log messages should now appear in your journal.
+
+### Testing the Python bridge
```

Comment:
Also let's add a section about the `print` tool:

https://github.com/cockpit-project/cockpit/issues/19193

Reply:
The above does that now, so resolving.